LeGO-LOAM/include/utility.h

```diff
@@ -153,6 +154,8 @@ POINT_CLOUD_REGISTER_POINT_STRUCT (PointXYZIRPYT,
                                    (double, time, time)
 )
 
+typedef Eigen::Vector3f Vector3D;
```

Comment:
Why typedef and not using?

Reply:
I am an old guy... Yes, it is equivalent.